docs/pricing/index.mdx

```diff
@@ -76,7 +76,35 @@ Tracing is enabled by and will be billed for in spans.
 
 #### Cron Monitors Pricing
 
-Monitors for Crons can't be purchased in advance. Crons are available to use and purchase through a pay-as-you-go budget that can be shared across all event types.
+All Sentry plans include **one cron monitor**. To activate additional monitors, set up a **Pay-As-You-Go (PAYG) budget**. Additional monitors can't be purchased in advance; they are only available through your PAYG budget, which can be shared across all event types.
```

Comment:
I believe the official stylization of payg is "pay-as-you-go" unless it's at the start of a sentence